services/data_checker.js

```diff
@@ -5,26 +5,26 @@ const uuidRegex = /^([0-9a-fA-F]{8})-(([0-9a-fA-F]{4}\-){3})([0-9a-fA-F]{12})$/i
 
 const checkUUID = (uuid) => uuidRegex.test(uuid)
 
-const checkMail = (mail) => mailRegex.test(mail.toLowerCase())
+const checkMail = (mail) => mail !== undefined && mail !== null ? mailRegex.test(mail.toLowerCase()) : false
```

Comment:
warunek można zastąpić przez `mail != null`

Reply:
Wydaje mi się, że nawet mail ? by zadziałało (plus wtedy mamy pewność, że nie ma pustego maila)

Reply:
plus wydaje mi się, że mógłbyś to zapisać
```js
mail && mailRegex.test(...)
```
ale tu się mogłem walnąć